examples/arkane/networks/acetyl+O2_sls/input.py

```diff
@@ -0,0 +1,284 @@
+        HinderedRotor(inertia=(9.35921,"u*angstrom**2"), symmetry=1, fourier=([[-11.2387,-12.5928,-3.87844,1.13314,-0.358812],[-1.59863,-8.03329,-5.05235,3.13723,2.45989]],"kJ/mol")),
+        HinderedRotor(inertia=(0.754698,"u*angstrom**2"), symmetry=1, barrier=(47.7645,"kJ/mol")),
+    ],
+    spinMultiplicity = 2,
+    opticalIsomers = 1,
+    frequency=(-404.271,'cm^-1'),
+)
+
+transitionState(
+    label = 'exit3',
+    E0 = (-7.2,'kcal/mol'),
+    modes = [
+        IdealGasTranslation(mass=(75.04,"g/mol")),
+        NonlinearRotor(inertia=([53.2821, 120.4050, 170.1570],"u*angstrom**2"), symmetry=1),
+        HarmonicOscillator(frequencies=([152.155,181.909,311.746,348.646,608.487,624.378,805.347,948.875,995.256,996.982,1169.1,1412.6,1834.83,3124.43,3245.2,3634.45],"cm^-1")),
+        HinderedRotor(inertia=(0.813269,"u*angstrom**2"), symmetry=1, fourier=([[-1.15338,-2.18664,-0.669531,-0.11502,-0.0512599],[0.00245222,0.0107485,0.00334564,-0.00165288,-0.0028674]],"kJ/mol")),
+    ],
+    spinMultiplicity = 2,
+    opticalIsomers = 1,
+    frequency=(-618.234,'cm^-1'),
+)
+
+################################################################################
+
+reaction(
+    label = 'entrance1',
+    reactants = ['acetyl', 'oxygen'],
+    products = ['acetylperoxy'],
+    transitionState = 'entrance1',
+    kinetics = Arrhenius(A=(2.65e6,'m^3/(mol*s)'), n=0.0, Ea=(0.0,'kcal/mol'), T0=(1,"K")),
+)
+
+reaction(
+    label = 'isom1',
+    reactants = ['acetylperoxy'],
+    products = ['hydroperoxylvinoxy'],
+    transitionState = 'isom1',
+    tunneling = 'Eckart',
+)
+
+reaction(
+    label = 'exit1',
+    reactants = ['acetylperoxy'],
+    products = ['ketene', 'hydroperoxyl'],
+    transitionState = 'exit1',
+    tunneling = 'Eckart',
+)
+
+reaction(
+    label = 'exit2',
+    reactants = ['hydroperoxylvinoxy'],
+    products = ['ketene', 'hydroperoxyl'],
+    transitionState = 'exit2',
+    tunneling = 'Eckart',
+)
+
+reaction(
+    label = 'exit3',
+    reactants = ['hydroperoxylvinoxy'],
+    products = ['lactone', 'hydroxyl'],
+    transitionState = 'exit3',
+    tunneling = 'Eckart',
+)
+
+#################################################################################
+
+network(
+    label = 'acetyl + O2',
+    isomers = [
+        'acetylperoxy',
+        'hydroperoxylvinoxy',
+    ],
+    reactants = [
+        ('acetyl', 'oxygen'),
+        ('ketene', 'hydroperoxyl'),
+	('lactone', 'hydroxyl')
+    ],
+    bathGas = {
+        'nitrogen': 1.0,
+    }
+)
+
+#################################################################################
+
+pressureDependence(
+    'acetyl + O2',
+    Tmin=(300.0,'K'), Tmax=(2000.0,'K'), Tcount=8,
+    Pmin=(0.01,'bar'), Pmax=(100.0,'bar'), Pcount=5,
+    maximumGrainSize = (1.0,'kcal/mol'),
+    minimumGrainCount = 250,
+    method = 'simulation least squares',
```

Comment:
I believe this is the same as asking for "simulation least squares matrix exponential"

Reply:
Ah ok, I will drop one of them